regression/validate-trace-xml-schema/check.sh

```diff
@@ -0,0 +1,29 @@
+if ! command -v python3 > /dev/null; then
+    echo "python3 not found, skipping XSD tests"
+    exit
+fi
+
+python_minor_version=$(python3 --version | cut -d ' ' -f 2 | cut -d '.' -f 2)
+
+if [ $python_minor_version -lt 5 ]; then
+    echo "python version less than 3.5, skipping XSD tests"
```

Comment:
:bulb: I think I would exit with a non-zero exit code here, to prevent silent failures on CI :ghost:

Reply:
This is not supposed to fail CI

Reply:
What I'm suggesting is it should fail CI (and the CIs where this doesn't work, we disable it in the configuration file). Currently if a config breaks in a way that means we have the wrong version of python, this test will just be disabled and we'll never know.